.gitlab/kernel_version_testing/system_probe.yml

```diff
@@ -348,6 +351,7 @@ kernel_matrix_testing_setup_env_x64:
   variables:
     AWS_EC2_SSH_KEY_FILE: $CI_PROJECT_DIR/ssh_key
     RETRY: 2
+    MICROVM_GREP_PATTERN: "$ARCH-$TAG-.*$TEST_SET.*-ddvm"
```

Comment:
Why the `.*`? shouldn't this be a strict sequence of values?

Reply:
The pattern is generated as `arch-tag-vmsetTags-ddvm-.*`. Since we are not enforcing any ordering on the vmset-tags we need the `.*`